docs/operate/control/api-keys.md

```diff
@@ -41,7 +41,7 @@ Click **Choose** to confirm your selection.
 
 ## Rotate an API key
 
-If you ever need to rotate an API key:
+If you ever need to rotate an API key you can use the [Viam app](https://app.viam.com/) or the [`RotateKey`](/dev/reference/apis/fleet/#rotatekey) method:
```

Comment:
```suggestion
If you ever need to rotate an API key you can use the [`RotateKey`](/dev/reference/apis/fleet/#rotatekey) method, or you can use the [Viam app](https://app.viam.com/) as follows:
```
Or move rotate key into a separate sentence at the bottom. The way it is now makes it seem like the steps below will use this method directly.

Reply:
```suggestion
If you ever need to rotate an API key you can use the [Viam app](https://app.viam.com/):
```